setup.py

```diff
@@ -122,6 +122,7 @@ def _read_file(filename):
         'zope.traversing',
         'zope.viewlet',
         'multipart',
+        'legacy-cgi; python_version > "3.12"',
```

Comment:
This is an experiment. The dependency must go into `zope.testbrowser` instead. See https://github.com/zopefoundation/zope.testbrowser/pull/108. This will need merging and then a release that is pulled in here so this dependency can go.

Reply:
Removed the dependency, it's now moved into zope.testbrowser itself.